USAGE_android.md

```diff
@@ -388,10 +780,43 @@ activity with the following:
 
 ```bash
 adb shell am force-stop com.lunarg.gfxreconstruct.replay
-adb shell am start -n "com.lunarg.gfxreconstruct.replay/android.app.NativeActivity" -a android.intent.action.MAIN -c android.intent.category.LAUNCHER --es "args" "<arg-list>"
+adb shell am start -n "com.lunarg.gfxreconstruct.replay/android.app.NativeActivity" \ 
+                   -a android.intent.action.MAIN \ 
+                   -c android.intent.category.LAUNCHER \ 
+                   --es "args" \ 
+                   "<arg-list>"
+```
+
+If `gfxrecon-replay` was built with Vulkan Validation Layer support,
+`VK_LAYER_KHRONOS_validation` can be enabled and disabled in the same manner as
+`VK_LAYER_LUNARG_gfxreconstruct`
+
+
+#### Using the Replay Command
+
+To replay a captured trace, find the location of the trace and replay it using
+the built `gfxrecon-replay` tool:
+
+```bash
+./android/scripts/gfxrecon.py replay ${capture file path}
+```
+
+For example, if the capture was saved to the `/sdcard/Download` folder as
+"gfxrecon_capture_20221210T120558.gfxr", the command would look like:
+
+```bash
+./android/scripts/gfxrecon.py replay /sdcard/Download/gfxrecon_capture_20221210T120558.gfxr
+```
+
+##### Pushing Local Replays
+
+If you have the capture file stored locally on your host machine and you want to
+push it to the target device as part of the replay, you can use the `-p` option:
+
+```bash
+./android/scripts/gfxrecon.py replay -p gfxrecon_capture_20221210T120558.gfxr
```

Comment:
Did you try this?  I haven't used it lately but I was under the impression, and the script usage message also says this, that you need to provide the local filename and the remote filename.

Reply:
Removed the whole "-p" option as it's not supported and removed from all other documentation.

Reply:
Ok, I'm completely blind.  It was there.  I was looking in the wrong location.  Updated with more detail.